src/main/java/com/synopsys/integration/alert/web/security/authentication/UserManagementAuthoritiesPopulator.java

```diff
@@ -107,17 +116,31 @@ public String getSAMLRoleAttributeName(String defaultName) {
         return roleMapping;
     }
 
+    private Set<String> getRolesFromDatabase(String userName, boolean appendRolePrefix) {
+        Function<String, String> function = appendRolePrefix ? (roleName) -> UserModel.ROLE_PREFIX + roleName : Function.identity();
+        Optional<UserModel> userModel = userAccessor.getUser(userName);
+        Set<String> roleNames = userModel.map(UserModel::getRoleNames).orElse(Set.of());
+        Set<String> newRoleNames = new LinkedHashSet<>(roleNames.size());
+        for (String roleName : roleNames) {
+            newRoleNames.add(function.apply(roleName));
+        }
+        return newRoleNames;
+    }
+
+    @Deprecated
```

Comment:
Do we need to deprecate private methods? I imagine that we just remove these methods when they're no longer used.